practice_05.11.19_classes/Task2_Plants/Plant.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace practice_05._11._19_classes.Task2_Plants
+{
+    public class Plant
+    {
+        public string Type { get; set; }
+
+        public double Height { get; set; }
```

Comment:
Height and Color is used inside this class only. So no need to make them public.

Reply:
fixed